typegate/src/runtimes/mod.ts

```diff
@@ -38,9 +35,18 @@ export async function initRuntime(
 }
 
 export async function init_runtimes(): Promise<void> {
-  for await (const file of Deno.readDir(localDir)) {
-    if (file.isFile && file.name.endsWith(".ts") && file.name !== "mod.ts") {
-      await import(resolve(localDir, file.name));
-    }
-  }
+  await Promise.allSettled([
```

Comment:
I believe a standard import would be cleaner as the dynamic loading does not work any more, e.g. to register a new runtime, you would need to link it here. I suppose we can even remove the automatic registration, and build manually the `registeredRuntimes` object. WYDT @Natoandro ?